tpcc/main.go

```diff
@@ -193,24 +203,22 @@ func main() {
 	start := time.Now()
 	errCh := make(chan error)
 	var wg sync.WaitGroup
-	if *noWait {
+	for i := range workers {
+		w := i % *warehouses
+		workers[i] = newWorker(i, w, dbs[w%len(dbs)], &wg)
+	}
+
+	go func() {
+		// Starter the workers evenly spaced over 30s to avoid any thundering
```

Comment:
Hah! I told you this was a good idea.

Reply:
We should make this configurable with a flag.

Reply:
Added a `-ramp` flag.